2020-04-29-fedora-31-template-available.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Fedora 30 approaching EOL, Fedora 31 TemplateVM available, Fedora 32 TemplateVM in testing"
+categories: announcements
+author: Andrew David Wong
+---
+
+This announcement includes several updates regarding Fedora TemplateVMs.
+
+## Fedora 30 approaching EOL
+
+With the release of Fedora 32 on April 21, Fedora 30 is expected to
```

Comment:
It was released yesterday: https://fedoramagazine.org/announcing-fedora-32/

Reply:
I found a different source say it was April 21. Very frustrating that they don't have it clearly documented in one official place.

Reply:
Looks like it was pushed from April 21 to April 28, and the F30 EOL date is set for May 26:

https://fedorapeople.org/groups/schedule/f-32/f-32-key-tasks.html